lambdas/process-journey-event/src/main/resources/statemachine/journey-maps/new-p2-identity.yaml

```diff
@@ -925,85 +750,91 @@ states:
         targetState: INELIGIBLE
 
   # Common Mitigation states - invalid-dl/invalid-passport
-  MITIGATION_PP_CRI_NINO:
+  MITIGATION_KBVS:
+    nestedJourney: KBVS
+    exitEvents:
+      fail-with-no-ci:
+        targetJourney: INELIGIBLE
+        targetState: INELIGIBLE
+      next:
+        targetJourney: EVALUATE_SCORES
+        targetState: START
+      enhanced-verification:
+        targetJourney: FAILED
+        targetState: FAILED
+
+  F2F_FAILED_MITIGATION_PAGE:
     response:
-      type: cri
-      criId: nino
-    parent: CRI_STATE
+      type: page
+      pageId: pyi-f2f-technical
     events:
       next:
-        targetState: MITIGATION_CRI_HMRC_KBV
-      fail-with-no-ci:
-        targetState: MITIGATION_PRE_EXPERIAN_KBV_TRANSITION_PAGE
+        targetState: MITIGATION_01_IDENTITY_START_PAGE
+        auditEvents:
+          - IPV_MITIGATION_START
+        auditContext:
+          mitigationType: enhanced-verification
+      end:
+        targetState: RETURN_TO_RP
 
-  MITIGATION_PRE_EXPERIAN_KBV_TRANSITION_PAGE:
+## Temporary inclusion of KBV states to avoid breaking existing journeys
```

Comment:
(ok this has messed up the diff more than I hoped - you might want to ignore the last commit!)